src/azul/health.py

```diff
@@ -48,13 +65,41 @@ def queues(self):
                 response[queue] = {
                     'up': True,
                     'messages': {
-                    'delayed': int(queue_instance['ApproximateNumberOfMessagesDelayed']),
-                    'invisible': int(queue_instance['ApproximateNumberOfMessagesNotVisible']),
-                    'queued': int(queue_instance['ApproximateNumberOfMessages'])
+                        'delayed': int(queue_instance['ApproximateNumberOfMessagesDelayed']),
+                        'invisible': int(queue_instance['ApproximateNumberOfMessagesNotVisible']),
+                        'queued': int(queue_instance['ApproximateNumberOfMessages'])
                     }
                 }
         return response
 
+    @memoized_property
+    def progress(self) -> JSON:
+        return {
+            'up': True,
+            'unindexed_bundles': sum(self.queues[config.notify_queue_name].get('messages', {}).values()),
+            'unindexed_documents': sum(self.queues[config.document_queue_name].get('messages', {}).values())
+        }
+
+    @memoized_property
+    def api_endpoints(self):
+        status = {'up': True}
+
+        for endpoint in self.endpoints:
+            response = requests.head(f'{config.service_endpoint()}{endpoint}')
+            try:
+                response.raise_for_status()
+            except requests.exceptions.HTTPError as e:
+                status[endpoint] = {
```

Comment:
I think we should make the URLs absolute:

```suggestion
                status[config.service_endpoint() + endpoint] = {
```